test/app-tap/datetime.test.lua

```diff
@@ -375,6 +379,95 @@ test:test("Formatting limits", function(test)
     end
 end)
 
+test:test("Simple tests for parser", function(test)
+    test:plan(4)
+    test:ok(date.parse("1970-01-01T01:00:00Z") ==
+            date.new{year=1970, mon=1, day=1, hour=1, min=0, sec=0})
+    test:ok(date.parse("1970-01-01T02:00:00+02:00") ==
+            date.new{year=1970, mon=1, day=1, hour=2, min=0, sec=0, tzoffset=120})
+
+    test:ok(date.parse("1970-01-01T02:00:00Z") <
+            date.new{year=1970, mon=1, day=1, hour=2, min=0, sec=1})
+    test:ok(date.parse("1970-01-01T02:00:00Z") <=
+            date.new{year=1970, mon=1, day=1, hour=2, min=0, sec=0})
+end)
+
+test:test("Multiple tests for parser (with nanoseconds)", function(test)
```

Comment:
Please, add some tests for `datetime.parse('value', {tzoffset=smth})`

Reply:
Yeah, that's omission now. Will handle.

Reply:
Added (please see comment with diff above in https://github.com/tarantool/tarantool/pull/6731/files#r827511211)